build.sh

```diff
@@ -31,6 +31,7 @@ HELP="$0 [clean] [libcudf] [cudf] [dask_cudf] [benchmarks] [-v] [-g] [-n] [-h]
    -v           - verbose build mode
    -g           - build for debug
    -n           - no install step
+   -x           - do not build nvtx
```

Comment:
I don't like `-x` for this. How about `--disable_nvtx`?

Reply:
You are right. That was not a smart flag name. Will change that. Not sure what I was thinking?

Reply:
Once you change this I can approve.